Stripe/STPLegalEntityParams.h

```diff
@@ -0,0 +1,48 @@
+//
+//  STPLegalEntityParams.h
+//  Stripe
+//
+//  Created by Daniel Jackson on 12/20/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "STPFormEncodable.h"
+
+@class STPAddress;
+
+
+@interface STPVerificationParams: NSObject<STPFormEncodable>
+@property (nonatomic, copy, nullable) NSString *document;
+@end
+
+
+@interface STPPersonParams: NSObject<STPFormEncodable>
+@property (nonatomic, copy, nullable) NSString *firstName;
+@property (nonatomic, copy, nullable) NSString *lastName;
+@property (nonatomic, copy, nullable) NSString *maidenName;
+@property (nonatomic, strong, nullable) STPAddress *address;
+@property (nonatomic, copy, nullable) NSDateComponents *dateOfBirth;
+@property (nonatomic, strong, nullable) STPVerificationParams *verification;
+@end
+
+
+typedef NS_ENUM(NSUInteger, STPLegalEntityType) {
+    STPLegalEntityTypeIndividual,
+    STPLegalEntityTypeCompany,
+};
+
+
+@interface STPLegalEntityParams : STPPersonParams
```

Comment:
Any problems foreseen by inheriting from `STPPersonParams`, which conforms to `STPFormEncodable`?

Reply:
hmm, I can't think of any